templates/include/metadata/ingestionCommonMetadata.1.schema.json

```
@@ -59,4 +59,26 @@
       "type": "string"
     }
   }
+},
+"isp": {
+  "type": "object",
+  "description": "Results of ISP lookup based on the client's IP address",
+  "properties": {
+    "db_version": {
+      "description": "The specific geo ISP database version used for this lookup",
+      "type": "string"
+    },
+    "name": {
+      "description": "The name of the ISP associated with the client's IP address",
+      "type": "string"
+    },
+    "organization": {
+      "description": "The name of the organization associated with the client's IP address",
```

"Organization" seems vague. Perhaps we can be a bit more specific based on the description in https://www.maxmind.com/en/geoip2-isp-database

```suggestion
      "description": "The name of a specific business entity associated with the client's IP address when available; otherwise the ISP name",
```